test-scripts/run-presto-kerberos-integration-test.sh

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env bash
+# runs one or more Metabase test(s) against a Kerberized Presto instance
+
+# install clojure version needed for Metabase
+curl -O https://download.clojure.org/install/linux-install-1.10.3.933.sh
+chmod +x linux-install-1.10.3.933.sh
+./linux-install-1.10.3.933.sh
```

Comment:
Why not just have the GH action itself do this stuff? That way you can still run this test script locally without it installing a bunch of stuff for you

Reply:
All of this is running within the container managed by `mba run`, and not on the "raw" metal GitHub action executor itself.  It was a couple months since I last touched this stuff, but mostly because the GitHub action extremely cumbersome and hard to debug the more steps that were added.

I could add some check to only do this if the proper version of Clojure isn't already installed.